jupyterlab-base/scripts/install-base-packages

```diff
@@ -1,23 +1,42 @@
 #!/usr/bin/env bash
 set -x
 
-# This script updates packages in the base Docker image that's used by both
-# the build and runtime images, and gives us a place to install additional
-# system-level packages with apt-get.
+# This script installs additional packages used by the dependency image but
```

Comment:
Since this now installs runtime packages the comment _not needed by the runtime image_ might not be accurate?